data/_models/shared/Organisations.csv

```diff
@@ -1,15 +1,15 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
 Organisations,,,,,,,,,,,"org:Organization,foaf:Agent",Research departments and research groups,"DataCatalogueFlat,EMA",
 Organisations,,overview,heading,,,,,,,,,General information,DataCatalogueFlat,
-Organisations,,id,,1,true,,,,,,dcterms:identifier,Internal identifier,"DataCatalogueFlat,EMA",
-Organisations,,pid,,2,,,,,,,dcterms:identifier,Persistent identifier,"DataCatalogueFlat,EMA",
-Organisations,,acronym,,,,,,,,,dcterms:identifier,Acronym if applicable,"DataCatalogueFlat,EMA",
-Organisations,,name,text,3,true,,,,,,"dcterms:title,foaf:name",Name used in European projects,"DataCatalogueFlat,EMA",
-Organisations,,type,ontology_array,,,CatalogueOntologies,Organisation types,,,,,Type of organisation; in which sector is the organisation active?,"DataCatalogueFlat,EMA",
+Organisations,,id,,1,true,,,,,,dct:identifier,Internal identifier,"DataCatalogueFlat,EMA",
```

Comment:
this is an internal id, should not be annotated

Reply:
this ID is used to build the real identifier IRI by adding server/schema path, I suppose?

Reply:
Indeed, internal id can be annotated that is not a problem at all. There is no fundamental rule that this should be a pid.